src/v2/api/services/PollService.ts

```diff
@@ -111,7 +110,7 @@ export class PollService {
     teacherId: string,
     query: CommentsQueryDTO = {}
```

Comment:
Add trailing comma. Also remove unnecessary `= {}`

Reply:
The {} is needed for questionData and commentsData because query.subjectId, query.year, ect will be undefined if query is {}. Otherwise query will be undefined and query.subjectId, query.year, ect won't work